src/wizard/Step1/components/PermissionsModal.vue

```diff
@@ -1,11 +1,52 @@
 <template>
-  <div>Permissions Modal</div>
+  <v-dialog
+    v-model="isDialogVisible"
+    max-width="600px"
+    persistent
+    class="portfolio-managers-modal"
+  >
+    <v-card>
+      <v-card-title> Invite Portfolio Manager </v-card-title>
+      <v-card-text>
+        <p>
+          Portfolio Managers can have different levels of access based on their
+          roles within your organization. Invite multiple people with the same
+          permissions at once.
+        </p>
+        <br />
+        <ATATTextField label="Email address"
+                       :messages="['Messages']">
+          Email address
+        </ATATTextField>
+        <br />
+        <v-btn class="link-body-md">Add another portfolio manager</v-btn>
+        <br /><br />
+      </v-card-text>
+      <v-card-actions>
+        <v-btn color="primary" text @click="dialog2 = false"> Close </v-btn>
+      </v-card-actions>
+    </v-card>
+  </v-dialog>
 </template>
 
 <script lang="ts">
 import Vue from "vue";
-import { Component } from "vue-property-decorator";
+import { Component, Prop } from "vue-property-decorator";
+import ATATTextField from "@/components/ATATTextField.vue";
 
-@Component({})
-export default class PermissionsModal extends Vue {}
+@Component({
+  components: { ATATTextField },
+})
+export default class PermissionsModal extends Vue {
+  @Prop() isDialogVisible = true;
+
+  private doCancel(): void {
+    // do something
```

Comment:
I emitted an event form the button nav bar component to the parent Wizard Component.   Easy to do. Am sure you know,  The code is there is you need to see it.